index.html

```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
       </ul>
     </section>
 
-    <footer>
+    <section class="recent-works">
+      <hxr class="works-title">
+        <h2>My Recent Works</h2>
+        <hr>
+      </hxr>
+      <section class="multi-stories ">
+        <img src="images/img_placeholder.png" alt="featured image">
+
+        <hxr class="right-block">
+          <h2>Multi-Post Stories</h2>
+          <p> A daily selection of privately personalized reads; no accounts or sign-ups required. 
+              has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a standard dummy text.</p>
+          
+          <hrx class="tags">
+            <ul>
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </hrx>    
+          <button type="button">See Project</button>
+        </hxr>
```

Comment:
- [ ] Please kindly check the style of this button, from the project requirements, you should stick to the design as much as possible. But as you can see you have some differences in the design. You can see the design below and compare it with your project design. :pray:

| Your Design (there is a line break in between the `see` and `project` |Figma Design (No line break in between the `see` and `project`) |
| :---:   | :---: | 
| ![Screenshot from 2022-10-28 21-10-24](https://user-images.githubusercontent.com/92303891/198733775-b7bc8761-7ac1-4083-b8ac-b6d0c13cf552.png) | ![Screenshot from 2022-10-28 21-14-53](https://user-images.githubusercontent.com/92303891/198734099-6be0b5c2-d7f3-4dea-a9d0-865b40d67c0a.png)
 |